src/docs/asciidoc/60-documentation.adoc

```diff
@@ -199,7 +199,8 @@ class DocumentationTests {
 }
 ----
 
-A canvas generated looks like this:
```

Comment:
Can we change this to “…to the `spring-modulith-docs` folder in your build system's build folder”? I'd like to avoid referring to individual build systems and their internals.

Reply:
done